app/integration/bom/pom.xml

```diff
@@ -20,17 +20,17 @@
 
   <groupId>io.syndesis.integration</groupId>
   <artifactId>integration-bom</artifactId>
-  <version>1.10-SNAPSHOT</version>
+  <version>2.0-SNAPSHOT</version>
   <name>Integration :: Bill of Materials (BOM)</name>
   <description>Syndesis :: Integration :: ${project.name}</description>
 
   <packaging>pom</packaging>
 
   <properties>
-    <spring-boot.version>2.1.6.RELEASE</spring-boot.version>
-    <camel.version>2.23.2.fuse-760024</camel.version>
-    <atlasmap.version>2.0.0-M.4</atlasmap.version>
-    <jackson.databind.version>2.9.9</jackson.databind.version>
+    <spring-boot.version>2.2.6.RELEASE</spring-boot.version>
+    <camel.version>3.1.0</camel.version>
+    <atlasmap.version>2.0.1</atlasmap.version>
```

Comment:
Unfortunately cherry-pick is not smart enough to pick this specific line up... spring/camel/jackson.databind needs to stay in 1.10.x.

Reply:
ok cool, wasn't sure about jackson.databind

Reply:
Does this look right now? I had to redo the backport because I didn't properly merge the commit from your PR

Reply:
Yep, it's PITN, you'd need to update it manually in this file, to update `atlasmap.version` only in 1.10.x because master has different version set for those. Other than atlasmap needs to stay same.